team-telnyx/webrtc
# feat(WEBRTC-107):  added support to chose servers dynamically

Added support to chose servers dynamically.

- You can choose between servers `development` or `production` with property `env`
- You can pass your own iceServers with property `iceServers`
- Updated examples to support `env`, `host` and `iceServers`


## 📝 To Do


## ✋ Manual testing

1. `npm run build` in root folder
2. Navigate into `examples/vanilla`
3. Open `index.html`
4. Click in connect and see in Network tab on WS option if is connected in host `rtc` (production host)
5. Uncheck production checkbox and click in connect and see in Network tab on WS option if is connected in host `rtcdev` (production host)

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |![image](https://user-images.githubusercontent.com/16343871/83880679-e1a41480-a715-11ea-965d-63536a51364c.png)|
